modules/converting-machine-api-infra-machines.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="converting-machine-api-infra-machines_{context}"]
+= Converting infrastructure machines created using the Machine API
+
+[IMPORTANT]
+====
+Creating infra machines using the Machine API is now deprecated.
+====
+
+Infrastructure machines created using the Machine API had both the `worker` and `infra` labels applied. This duel label required you to taint the node so that compute workloads were not scheduled on it. Managing infra workloads using taints is no longer recommended. Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool manage it.
+
+.Procedure
+
+. List the infra nodes available in the cluster:
++
+[source,terminal]
+----
+$ oc get nodes -l node-role.kubernetes.io/infra
+----
++
+.Example output
+[source,terminal]
++
+----
+NAME                           STATUS   ROLES          AGE   VERSION
+ip-10-0-205-158.ec2.internal   Ready    worker,infra   64m   v1.20.0+551f7b2.20.0
+----
+
+. Remove the `worker` label from the node you want to use as an infra node:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-
+----
+
+. Create a dedicated machine config pool to manage your infra nodes.
+
+[NOTE]
+====
+When removing the `worker` label from your infra node, it is no longer managed by the default machine config pool configured for compute nodes; it is unmanaged. This means the Machine Config Operator (MCO) no longer manages the node, and can not be upgraded. A dedicated machine config pool for your infra nodes allows the MCO to manage the lifecycle of those node types.
```

Comment:
We need to link to https://github.com/openshift/openshift-docs/pull/30683/files#diff-2901799cc6024aefc0d385a7a42a24f967dc1d1e2b9a4a349b3529f39541e966 here. 

I wonder if this needs to be done first, before doing these steps?

Reply:
Agreed that there needs to be a link here to the procedure describing how to do create a dedicated machine config pool for infrastructure nodes.

Reply:
Due to a tooling deficiency we have with providing xrefs in a module, I cannot add a link in the file. However, I did add one to the assembly [here](https://github.com/openshift/openshift-docs/pull/30683/files#diff-baff329b87e4a2014b04aaae44e70db65b9a03833bb0c4c2a338eb77afc93c19R27) directly after the module. So there is a link provided after this note when the docs are rendered.